packages/cli/src/commands/init.ts

```diff
@@ -51,60 +45,20 @@ export default class Init extends BaseCommand {
     }
   }
 
-  async promptMetaData(yes: boolean): Promise<Meta> {
-    const answer: any = yes
-      ? { name: 'My Awesome Tutorial' }
-      : await prompt([
```

Comment:
I think the `name` prompt should be kept.

Reply:
@mRcfps Good idea!